varia/sass/abstracts/_responsive-logic.scss

```diff
@@ -131,26 +131,31 @@ $content-width-xxl: calc( #{map-deep-get($config-global, "breakpoint", "xxl")} -
 	@include media(mobile) {
 		/*rtl:ignore*/
 		margin-right: calc( 0.5 * (100vw - #{$content-width-sm}) );
+		max-width: calc( 0.5 * #{$content-width-sm} );;
```

Comment:
Let’s add a margin to the left to prevent these blocks from getting too close to the sibling blocks. (Also, please remove that double `;` there).

```suggestion
		max-width: calc( 0.5 * #{$content-width-sm} );
		/*rtl:ignore*/
		margin-left: #{map-deep-get($config-global, "spacing", "horizontal")};

```

Before|After
------------|------------
![image](https://user-images.githubusercontent.com/709581/85317485-73d54800-b48c-11ea-9f1d-9167994e3e3e.png)|![image](https://user-images.githubusercontent.com/709581/85317538-83549100-b48c-11ea-976d-b0f0bd3a9841.png)